.circleci/config.yml

```diff
@@ -21,6 +21,23 @@ workflows:
                       branches:
                           only:
                               - /^(release\/v\d+\.\d+\.\d+|hotfix\/v\d+\.\d+\.\d+|master|develop)$/
+    nightly_build:
+        triggers:
+            - schedule:
+                cron: "0 21 * * *"
+                filters:
+                    branches:
+                        only:
+                            - master
```

Comment:
I think this is false now, since if I remember correctly this would now disable the commit trigger for master branch, and wasn't the idea that there is a separate nigthly-build branch against which these tests would be ran, not master?